XenAdmin/Wizards/ImportWizard/ImportWizard.cs

```diff
@@ -251,17 +251,30 @@ protected override void UpdateWizardContent(XenTabPage page)
                     m_pageImportSource.DiskCapacity, m_pageImportSource.IsWIM, m_pageVMconfig.AdditionalSpace,
                     m_pageImportSource.FilePath, m_pageImportSource.ImageLength);
 
+				if (m_pageStorage.SelectedOvfEnvelope != m_envelopeFromVhd

```

Comment:
An easier way to do this is assign the newly created envelope to a local variable and compare this with `m_envelopeFromVhd` before assigning to the latter:
```
var newEnv = OVF.Create(...);
if (newEnv != m_envelopeFromVhd) {...}
m_envelopeFromVhd = newEnv;
```